modules/floating-buttons/classes/render/floating-buttons-core-render.php

```diff
@@ -50,4 +51,13 @@ protected function add_layout_render_attribute( $layout_classnames ) {
 		] );
 	}
 
+	protected function add_content_wrapper_render_attribute() {
+		$this->widget->add_render_attribute( 'content-wrapper', [
+			'aria-hidden' => 'true',
+			'aria-label' => __( 'Links window', 'elementor' ),
+			'class' => 'e-contact-buttons__content-wrapper hidden',
+			'id' => uniqid( 'e-contact-buttons__content-wrapper-' ),
```

Comment:
@willhowat this is for you

Reply:
@louiswol94 this related to the comment above https://github.com/elementor/elementor/pull/27687#discussion_r1644608433

The ID is required for aria-controls attribute.
At the time of implementing this and it may still be the case, it was possible to add multiple floating button instances which required the IDs be unique

Reply:
Can we use the `$this->get_id_int()` methods here?